docs/pages/tutorial/gestures.mdx

```diff
@@ -234,14 +228,7 @@ Let's take a look at our app on iOS, Android and the web:
 
 A pan gesture allows recognizing a dragging gesture and tracking its movement. We will use this gesture handler to drag the sticker across the image.
 
-In the **EmojiSticker.js**, import `PanGestureHandler` from the `react-native-gesture-handler` library.
-
-{/* prettier-ignore */}
-```jsx EmojiSticker.js
-import { /* @info */ PanGestureHandler,/* @end */ TapGestureHandler} from "react-native-gesture-handler";
-```
-
-Create an `<AnimatedView>` component using the `Animated.createAnimatedComponent()` method. Then use it to wrap the `<TapGestureHandler>` component by replacing the `<View>` component.
+In the **EmojiSticker.js**, create an `<AnimatedView>` component using the `Animated.createAnimatedComponent()` method. Then use it to wrap the `<GestureDetector>` component by replacing the `<View>` component.
```

Comment:
Let's use built-in `Animated.View` from Reanimated instead of calling `Animated.createAnimatedComponent(View)` manually.

Also, do we want to use `Animated.Image` as well or shall we stick with `AnimatedImage` (so that we can substitute it with `Image` from `expo-image` for which Reanimated doesn't provide a built-in wrapped component)?

Reply:
It has some didactic value to show `Animated.createAnimatedComponent()`but I agree that it shouldn't be used on built-in `View` and `Image` components. They are exported from Reanimated under `Animated.View` and `Animated.Image` which is both shorter and simpler to use. 

cc @amandeepmittal what do you think?

Reply:
I think this is a good idea! I wasn't aware that Reanimated can export `View` and `Image` components. Can we stick to using `Image` from React Native instead of Expo Image since Expo Image doesn't work on Snack and running the examples from this tutorial is an integral part of this experience.